airbyte-integrations/connectors/destination-langchain/destination_langchain/config.py

```diff
@@ -110,11 +110,11 @@ def resolve_refs(schema: dict) -> dict:
         # config schemas can't contain references, so inline them
         json_schema_ref_resolver = RefResolver.from_schema(schema)
         str_schema = json.dumps(schema)
-        for ref_block in re.findall(r'{"\$ref": "#\/definitions\/.+?(?="})"}', str_schema):
+        for ref_block in re.findall(r'{"\$ref": "#\/\$defs\/.+?(?="})"}', str_schema):
```

Comment:
I like how diligent you are to have caught such a specific case! 

Should we re-use [the method in utils](https://github.com/airbytehq/airbyte/pull/39497/files#diff-720a7a97eb146ea2fa7b2b076f0c7bf177fabb7a40dcc265a858bc7e43f4eb1eR11)? It feels like when one change, the other will also change and therefore, they should have the same implementation. I would have easily seen myself missing this change